catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -41,23 +42,30 @@
         Function<T, Serializable> accessor = attribute.getAccessorFunction();
         Serializable value = accessor.apply(segment);
 
-        AttributeDescriptor descriptor = attribute.getAttributeDescriptor();
+        Set<AttributeDescriptor> descriptors = attribute.getAttributeDescriptors();
 
-        if (descriptor == null) {
+        if (descriptors == null) {
             LOGGER.debug("Could not set metacard attribute " + attribute.getLongName()
                     + " since it does not belong to this metacard type");
             return;
         }
 
-        if (descriptor.getType().equals(BasicTypes.STRING_TYPE) && value != null
-                && ((String) value).length() == 0) {
-            value = null;
+        for (AttributeDescriptor descriptor : descriptors) {
+            if (descriptor.getType()
+                    .equals(BasicTypes.STRING_TYPE) && value != null
+                    && value.toString().length() == 0) {
```

Comment:
Why value.toString().length()?  If you're already checking the Attribute type, can you just use `StringUtils.isNotEmpty()` here?